sdk/nodejs/tests/runtime/tsClosureCases.ts

```diff
@@ -4801,7 +4801,7 @@ return function () { console.log(o.b()); console.log(o.b.name); };
             expectText: `exports.handler = __f0;
 
 var __o2 = {};
-var __o2_b = {d: 3, c: 2};
+var __o2_b = {c: 2, d: 3};
```

Comment:
note: this is equivalent to before. t he change in behavior is due to how we now clear and re-add.  because of that, we restore this to the same order as the original object (i.e. 'c' then 'd', instead of 'd' then 'c').